docs/antora/modules/ROOT/pages/Getting_Started.adoc

```diff
@@ -2,26 +2,28 @@
 
 == Simple Project
 
-The simplest Mill build for a Java project looks as follows:
+The simplest Mill build for a Scala project looks as follows:
 
 .`build.sc`
-[source,scala]
+[source,scala,subs="attributes,verbatim"]
 ----
 import mill._, scalalib._
 
-object foo extends JavaModule {}
+object foo extends ScalaModule with RootModule{
```

Comment:
Nit: Should we also mention here, that `RootModule` is cosmetics for a single-module project, or link to that section? I mean, this is not the "simplest", as we could leave `RootModule`.

Also formatting...

```suggestion
object foo extends ScalaModule with RootModule {
```

Reply:
@lefou I think leaving it as-is is fine; in the next section, we immediately go into a multi-module example without a `RootModule`, which should be enough to introduce the "you don't use a root module" scenario to the user. Together, these examples should be enough to give a reasonable understanding to get started